schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonRequest.avdl

```diff
@@ -137,4 +141,29 @@ Types of files:
         */
         union {null, string} additionalDescription;
     }
+
+    /**
+    Interpretation flags at the participant level
+    */
+    record ParticipantInterpretationFlags{
+        /**
+        The interpretation flag
+        */
+        InterpretationFlags  interpretationFlag;
+        /**
+        The description for the flag
+        */
+        union {null, string} additionalDescription;
```

Comment:
If `InterpretationFlags` contains many flags, the `additionalDescription` then does not refer to a flag but to all. Isn't this a bit strange @Antonior26 ?